test/index.js

```diff
@@ -0,0 +1,125 @@
+const fs = require('fs');
+const path = require('path');
+const {expect} = require('chai');
+let hb;
+
+before(async function () {
+  hb = await require('..');
+});
+
+afterEach(function () {
+  if (this.blob) this.blob.destroy();
+  if (this.face) this.face.destroy();
+  if (this.font) this.font.destroy();
+  if (this.buffer) this.buffer.destroy();
+  this.blob = this.face = this.font = this.buffer = null;
+});
+
+describe('Face', function () {
+  it('collectUnicodes reflects codepoints supported by the font', function () {
+    this.blob = hb.createBlob(fs.readFileSync(path.join(__dirname, 'fonts/noto/NotoSans-Regular.ttf')));
```

Comment:
Can we reuse the fonts that are currently in `examples/` instead of adding new ones?